### ISSUE TITLE
'Append ROI data to features' is not functioning

### ISSUE THREAD

**Issue by onurserce** [sgoldenlab/simba#14]
**Describe the bug**
'Append ROI data to features' is not functioning as expected

**To Reproduce**
After successful execution of 'Analyze ROI' and 'Visualize ROI' steps:
1. Load the project
2. Go to 'Extract features' tab, click on 'Extract features', wait until 'All feature extraction complete.' message
3. Click 'Append ROI data to features'

**Screenshots**
![Bug](https://user-images.githubusercontent.com/31575364/77113850-ccc7b700-6a2b-11ea-8ec8-b6fdc33ac06c.PNG)
![Bug2](https://user-images.githubusercontent.com/31575364/77113855-ce917a80-6a2b-11ea-9539-0206ebcf1f40.PNG)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Comment by inoejj** [comment]
Thank you for the in depth report. In your anaconda prompt, can you type in `import pandas as pd`, then `pd.__version__`, and let me know what version of pandas you have installed?

If the version is not 0.25.3, please uninstall your pandas and reinstall pandas--version 0.25.3.

**Comment by onurserce** [comment]
Hi @inoejj,

My `pandas.__version__` seems to be 0.25.3 as required. The problem is arising from another source. 

Could that be because of numpy or something else?

Attached you can find all packages and their versions in my environment obtained with the command `conda list`

[packages.txt.txt](https://github.com/sgoldenlab/simba/files/4359937/packages.txt.txt)

**Comment by onurserce** [comment]
@inoejj @sronilsson 

Problem is caused by different column names in following two files:
...\project_folder\logs\measures\pose_configs\bp_names**\project_body_names.csv** and
...\project_folder\csv\features_extracted\extracted_features.csv

project_body_names.csv by default names body parts without any numbers (except Center_2, I selected 1 animal with 8bps throughout the whole project). However, SimBA generates the extracted features csv file such that it inserts numbers to each body part (either 1 or 2), probably owing to the fact that SimBA was initially written for tracking 2 animals and features were calculated based on these. 

The problem is solved by editing the project_body_names.csv file and appending `_1` to all body part names (except Center, it was changed from Center_2 to Center_1).
